tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -78,9 +78,9 @@ $ sudo pip3 install --upgrade https://storage.googleapis.com/tensorflow/mac/tens
 ```
 
 
-You can now [test your installation](#test_install).
+You can now [test your installation](#test-the-tensorflow-installation).
 
-## Virtualenv installation {#virtualenv_install}
```

Comment:
Sounds good. Do you have the exact transformation for the auto generated
anchors? I think it's lower(), strip(), s/[^a-z ]*//g, s/ */-/g

But I may be missing something subtle.
On Wed, Dec 30, 2015 at 14:54 Sam Abrahams <notifications@github.com> wrote:

> In tensorflow/g3doc/get_started/os_setup.md
> <https://github.com/tensorflow/tensorflow/pull/648#discussion_r48637615>:
>
> >
> > -## Virtualenv installation {#virtualenv_install}
>
> Sounds good. Right now I'm writing the script to do two things:
>
>    1. Remove the {#...} tags
>    2. Replace all of the old Markdown inline links to existing {#...}
>    tags and change them to the auto-generated style. i.e. check for any text
>    similar to [text](#oldheader) and changing it to
>    [text](#new-auto-generated-header)
>
> Should get this done by early tomorrow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/648/files#r48637615>.
>